ext/standard/basic_functions.c

```diff
@@ -115,6 +115,12 @@ PHPAPI php_basic_globals basic_globals;
 #include "php_fopen_wrappers.h"
 #include "streamsfuncs.h"
 #include "zend_frameless_function.h"
+#ifdef HAVE_CRYPT_H
```

Comment:
Shouldn't this be
```suggestion
#ifdef HAVE_CRYPT
```

Reply:
it is the same condition copy/paste from the one used in crypt.c